### DISCUSSION TITLE
Add a feature that can lock mods for a certain time. (to prevent player from playing them)

### DISCUSSION THREAD

**Discussion by Leclowndu93150** [ppy/osu#20647] [Ideas] [answered]
I have a really bad DT addiction and i cant help myself, i'd like something like a feature that can lock DT or any other mod for a certain time given by the player. for example, imagine i want to stop playing DT for 2 month, i just have to use that feature to lock the mod for 2 month then i would be forced to learn the other mods. You see ?

**Comment by WindowsMeosu** [comment]
I don't know about this one... would you have the ability to unlock any mods you apparently locked? what if you accidently locked a mod for a really long time? I am really unsure about this one here.

**Comment by Walavouchey** [comment]
idk, seems like a will issue. just don't play the mod? or maybe the game entirely? (there *is* such a thing as [locking yourself out of your account](https://osu.ppy.sh/wiki/en/Help_centre/Account#account-lock))

**Comment by peppy** [comment, marked as answer]
Out of scope sorry. We may add the ability to hide certain mods from the select screen, but it would not be timed.

**Reply by Leclowndu93150** [reply]
thank you so much
